### PR TITLE
[SYCL] [FPGA] Create latency control E2E emulator tests

### PR DESCRIPTION
**Pull request by shuoniu-intel**
Create E2E emulator tests for newly added latency control headers.

The corresponding SYCL PR: https://github.com/intel/llvm/pull/5066

### CODE REVIEW

**Comment by romanovvlad** [comment]
@MrSidims  @mlychkov  Could you please reivew?

**Review by MrSidims — DISMISSED** [review]

**Comment by shuoniu-intel** [comment]
@romanovvlad Could you please review and merge?

**Comment by romanovvlad** [comment]
> @romanovvlad Could you please review and merge?

I cannot merge, the pre-commit fails, maybe @vladimirlaz can help.

**Review by vladimirlaz — DISMISSED** [review]

**Comment by vladimirlaz** [comment]
@romanovvlad your approve is needed to enable merge button. CI failure should not block merge.

File: SYCL/Basic/fpga_tests/fpga_latency_control_lsu.cpp
Comment (romanovvlad) [inline SYCL/Basic/fpga_tests/fpga_latency_control_lsu.cpp:2]:
```suggestion
// REQUIRES: aoc, accelerator
// RUN: %clangxx -fsycl -fintelfpga %s -o %t.out
```
?

File: SYCL/Basic/fpga_tests/fpga_latency_control_pipe.cpp
Comment (romanovvlad) [inline SYCL/Basic/fpga_tests/fpga_latency_control_pipe.cpp:2]:
```suggestion
// REQUIRES: aoc, accelerator
// RUN: %clangxx -fsycl -fintelfpga %s -o %t.out
```
?

**Review by romanovvlad — APPROVED** [review]

**Review by vladimirlaz — APPROVED** [review]

**Pull request merged**
